discord-bot/src/commands/embedLink/index.ts

```diff
@@ -30,11 +35,11 @@ const embedLink = async (msg: Message) => {
   if (author.bot) return; // return if bot sends the message
 
   const currentTextChannel = channel as TextChannel;
-  let webhook = await fetchWebhook(currentTextChannel);
-  if (!webhook) {
-    webhook = await createWebhook(currentTextChannel); // create webhook if not found
-  }
-  if (!webhook) return; // return if can't find or create webhook
+  const webhook = await fetchOrCreateWebhook(
+    currentTextChannel,
+    'VAIT-EmbedLink-Hook'
```

Comment:
should we use a generic name for our webhook? Individual webhook for each function is redundant.